packages/mdc-switch/README.md

```diff
@@ -70,17 +74,66 @@ Users can add the `disabled` attribute directly to the `<input>` element or a pa
 CSS Class | Description
 --- | ---
 `mdc-switch` | Mandatory, for the parent element.
-`mdc-switch__native-control` | Mandatory, for the input checkbox.
-`mdc-switch__background` | Mandatory, for the background element.
-`mdc-switch__knob` | Mandatory, for the knob element.
+`mdc-switch__track` | Mandatory, for the track element.
+`mdc-switch__thumb-underlay` | Mandatory, for the ripple effect.
+`mdc-switch__thumb` | Mandatory, for the thumb element.
+`mdc-switch__native-control` | Mandatory, for the hidden input checkbox.
+
+## Style Customization
```

Comment:
There's already a style customization heading above, I think this one shouldn't be here

Reply:
Thanks, will remove.